src/ext/swap-errors.js

```diff
@@ -0,0 +1,9 @@
+htmx.defineExtension("swap-errors", {
+  onEvent: function (name, evt) {
+    if (name === "htmx:beforeSwap") {
+      if (evt.detail.xhr.getResponseHeader("HX-SWAP-ERRORS") !== null) {
```

Comment:
Only change I would make is to not use all caps, which is more consistent w/ other htmx headers.